zeeguu/core/feed_handler/feed_handler.py

```diff
@@ -3,13 +3,15 @@
 
 
 class FeedHandler:
-    def __init__(self, url: str, feed_type: int):
+    def __init__(self, url: str, feed_type: int, is_stored_db: bool = False):
         self.url = url
         self.feed_type = feed_type
         self.title = ""
         self.description = ""
         self.image_url_string = ""
-        self.extract_feed_metadata()
+        if not is_stored_db:
```

Comment:
There is something strange about this pattern. 
Could we leave the constructor to be just simply initializing the attributes with their default values, and then for those parts of the code where you would call `__init__` with `is_stored_in_db=True` call the `new_feed.extract_feed_metadata()`?

Reply:
I like that! Sure, we can do that!

Reply:
Tak!